modules/telemetry/events/status-events.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace VIPWorkflow\Modules\Telemetry\Events;
+
+use VIPWorkflow\Modules\Telemetry\Tracker;
+use WP_Post;
+
+function record_custom_status_change(
+	string $new_status,
+	string $old_status,
+	WP_Post $post,
+	Tracker $tracker
+): void {
+	if ( 'post' !== $post->post_type ) {
```

Comment:
We should also check `get_supported_post_types()` here.

Reply:
Done in 5752761a3e7e03a51257369550656b7e257602db